api/pom.xml

```diff
@@ -253,11 +253,25 @@
                     <windowtitle>Jakarta XML Registries ${project.version} API</windowtitle>
                 </configuration>
                 <executions>
+                    <execution>
+                        <id>attach-javadocs</id>
+                        <goals>
+                            <goal>jar</goal>
+                        </goals>
+                    </execution>
                     <execution>
                         <phase>javadoc</phase>
                         <goals>
                             <goal>javadoc</goal>
                         </goals>
+                        <configuration>
+                            <groups>
+                                <group>
+                                    <title>XML Registries API Documentation</title>
+                                    <packages>javax.management.j2ee</packages>
```

Comment:
Forgot to update the \<packages> when you did the copy-and-paste...  :-)

Reply:
Fixed now.